index.js

```diff
@@ -1,159 +1,98 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "extends": "stylelint-config-standard",
-    "customSyntax": "scss",
-    "plugins": [
-        "stylelint-scss"
-    ],
-    "ignoreFiles": [
-        "sass/vendor/*.scss"
-    ],
-    "rules": {
-        "at-rule-empty-line-before": [
-            "always",
+    'extends': 'stylelint-config-standard-scss',
+    'customSyntax': 'postcss-scss',
+    'rules': {
+        'alpha-value-notation': 'number',
+        'at-rule-empty-line-before': [
+            'always',
             {
-                "except": [
-                    "blockless-after-same-name-blockless",
-                    "first-nested"
-                ],
-                "ignore": [
-                    "after-comment"
-                ],
-                "ignoreAtRules": [
-                    "extend",
-                    "include",
-                    "else",
-                    "elseif",
-                    "content"
+                except: ['blockless-after-blockless', 'first-nested'],
+                ignore: ['after-comment'],
+                ignoreAtRules: [
+                    'media',
+                    'return',
+                    'extend',
+                    'include',
+                    'else',
                 ]
             }
         ],
-        "at-rule-no-unknown": [
```

Comment:
is this not longer required` we should add a test file without this `@rules` to make sure they work in our setup.

Reply:
@alexander-schranz This is no longer required as one of the extended sets handles this correct:

```
		'at-rule-no-unknown': null,
		'scss/at-rule-no-unknown': true,
```

I've tested it and it works as expected.